Handling Presentational role conflict resolution (#1189)
act-rules/act-rules.github.io: _rules/autocomplete-valid-value-73f2c2.md

## Accessibility Support

> - While `autocomplete` in a promising technique for supporting personalization in HTML, support for this is fairly limited.

Reviewer: ```suggestion
- Support for the `autocomplete` technique is fairly limited.
```
Just to remain as objective as possible. I don't see how "promising" is relevant here.

Author: I think this is a separate topic and should be part of another PR if needed. Would you like to handle it, @daniel-montalvo ?

#### Passed Example 8

> This `input` element has an [explicit role][] of `none`. However, it is [focusable][] (by default). Thus it has a [semantic role][] of `textbox` due to [Presentational Roles Conflict Resolution][]. It has a single autocomplete term.

Reviewer: This example does not have a label. Should we be adding it?

Author: This is topic for another PR, and much more generic than this specific rule.
See #1215.
